subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1232,7 +1237,11 @@ uint8_t ll_adv_enable(uint8_t enable)
 			/* schedule auxiliary PDU after primary channel PDUs */
 			ticks_anchor_aux =
 				ticks_anchor + ticks_slot +
-				HAL_TICKER_US_TO_TICKS(EVENT_MAFS_US);
+				HAL_TICKER_US_TO_TICKS(
+					MAX(EVENT_MAFS_US,
+					    EVENT_OVERHEAD_START_US) -
+					EVENT_OVERHEAD_START_US +
+					(EVENT_TICKER_RES_MARGIN_US << 1));
```

Comment:
Please put a comment why do you add `(EVENT_TICKER_RES_MARGIN_US << 1));` to the anchor point for auxiliary PDU.

Reply:
done.